react/orders101/src/features/header/Header.jsx

```diff
@@ -0,0 +1,21 @@
+function Header() {
+  return (
+    <div class="account container">
+      <ul class="account__sub-links">
+        <li>
```

Comment:
This would be another example where we could iterate over the data building up the header but since it's such a small and simple component I think it's fine like this too for now. 

We did speak about possibly extracting all texts and data into separate files containing those mocks. That would help with supporting a structure where the content is configurable and possibly support multiple languages. So if/when we get to that, then that could be an opportunity for dividing this component into smaller parts.

Reply:
iterated over mock data and passed data in new component